Userland/Applications/Presenter/PresenterSettings.cpp

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2022, kleines Filmröllchen <filmroellchen@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "PresenterSettings.h"
+#include "LibConfig/Client.h"
+#include <Applications/Presenter/PresenterSettingsFooterGML.h>
+#include <LibGUI/TextBox.h>
+
+PresenterSettingsFooterWidget::PresenterSettingsFooterWidget()
+{
+    load_from_gml(presenter_settings_footer_gml);
+    m_override_footer = find_descendant_of_type_named<GUI::CheckBox>("override_footer");
+    m_enable_footer = find_descendant_of_type_named<GUI::CheckBox>("enable_footer");
+    m_footer_text = find_descendant_of_type_named<GUI::TextBox>("footer_text");
+
+    m_override_footer->on_checked = [this](auto) { this->on_footer_settings_override_change(); };
```

Comment:
Please call `set_modified(true);` in this event listener, and add change listeners for the other widgets which also call it. That will make the "Apply" button and "..." in the close button work correctly.